task/02-numbers-tasks.js

```diff
@@ -202,7 +202,8 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    for (let i = 2; i < n; i += 1) if (n % i === 0) return false;
```

Comment:
please use a newline and braces for every logical step in the future, this is more readable